style(coral): Add custom theme

programmiri opened the pull request:
# About this change - What it does

- Adds custom klaw theme with DS variables (recommended way)

Resolves: #1312


### Screenshots 
I tried to make screenshots from all views.

<img width="1148" alt="Screenshot 2023-06-12 at 18 04 40" src="https://github.com/aiven/klaw/assets/943800/25751291-6cfb-4759-a7e0-6247b3140194">

<img width="1147" alt="Screenshot 2023-06-12 at 18 05 02" src="https://github.com/aiven/klaw/assets/943800/eaf980cb-0fe3-46d7-9597-043d86c4e0b2">

<img width="766" alt="Screenshot 2023-06-12 at 18 05 31" src="https://github.com/aiven/klaw/assets/943800/e8c997b4-630a-4f72-9521-f245d8da29c7">

<img width="825" alt="Screenshot 2023-06-12 at 18 05 15" src="https://github.com/aiven/klaw/assets/943800/72891d39-7386-454f-8619-e60cd6295f9a">

<img width="1405" alt="Screenshot 2023-06-12 at 18 06 52" src="https://github.com/aiven/klaw/assets/943800/513d4b62-ea29-4d89-bbcc-97c80eb0212e">

<img width="795" alt="Screenshot 2023-06-12 at 18 18 41" src="https://github.com/aiven/klaw/assets/943800/016b9d7e-fb42-4fa9-b4b4-e9d87a459974">

<img width="1105" alt="Screenshot 2023-06-12 at 18 13 25" src="https://github.com/aiven/klaw/assets/943800/64b31b7b-026e-4837-a1d3-8b6e4b8d2ed3">

<img width="1195" alt="Screenshot 2023-06-12 at 18 20 16" src="https://github.com/aiven/klaw/assets/943800/eceea2f1-3841-42cc-abca-c2193bd7da39">
trie
<img width="953" alt="Screenshot 2023-06-12 at 18 19 39" src="https://github.com/aiven/klaw/assets/943800/838cd0c2-723b-4fb0-abb5-bff26c9d1b1d">


<img width="761" alt="Screenshot 2023-06-12 at 18 24 55" src="https://github.com/aiven/klaw/assets/943800/6f0f5475-48af-40c7-ab42-b300e191a80c">

mathieu-anderson reviewed (approved):
It is so blue :o And overwriting css variables seem like the correct way to do theming, happy to see `aquarium` do it this way ^^

mathieu-anderson reviewed (changes requested):
But after running it locally and seeing the CI fail, there are some test failures 🤔 
<img width="505" alt="Screenshot 2023-06-13 at 09 18 38" src="https://github.com/aiven/klaw/assets/20607294/49d40b9d-c9a0-4fc1-9562-00de17fe2395">

programmiri commented:
❓ ❓ ❓  how can there be test failing 😭  Will check it out!

programmiri commented:
😭  it was me. I tried around with a potential different styling in Modal and added a "HU" to the text to see that I'm at the right button... and forgot to remove it 🤦‍♀️ 

Tests, again, saved me ❇️ 🙇 

Removed, all ✅  locally and pushed the change!

mathieu-anderson reviewed (approved):
Nice, thank you ^^
